### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by Myrausman**

### CODE REVIEW

File: src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java [hunk 1]
```diff
@@ -21,82 +21,71 @@
...
+    private ExpensesTrackerRepository expensesTrackerRepository;
 
     @GetMapping("/")
-    public ModelAndView index(Model model){
+    public ModelAndView index() {
         logger.debug("root to GET index");
         ModelAndView modelAndView = new ModelAndView("index");
-        List<ExpensesTrackerItem> expensesTrackerItems = new ArrayList<>();
-        ExpensesTrackerRepository.findAll().forEach(expensesTrackerItems::add);
+        List<ExpensesTrackerItem> expensesTrackerItems = (List<ExpensesTrackerItem>) expensesTrackerRepository.findAll();
```
Comment (samadpls) [inline src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java]: What the advantages of using list over array?

File: src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java [hunk 2]
```diff
@@ -21,82 +21,71 @@
...
-    public String createExpensesTrackerItem(@Valid ExpensesTrackerItem expensesTrackerItem, BindingResult result, Model model) {
+    public String createExpensesTrackerItem(@Valid ExpensesTrackerItem expensesTrackerItem, BindingResult result)
+    {
         if (result.hasErrors()) {
             return "add-ExpensesTracker-item";
         }
         expensesTrackerItem.setCreatedDate(Instant.now());
         expensesTrackerItem.setModifiedDate(Instant.now());
-        ExpensesTrackerRepository.save(expensesTrackerItem);
+        expensesTrackerRepository.save(expensesTrackerItem);
```
Comment (samadpls) [inline src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java]: Why e is small? Is it working?

File: src/main/java/com/scad/expensestracker/models/ExpensesTrackerItem.java [hunk 3]
```diff
@@ -2,58 +2,52 @@
...
     private Instant createdDate;
-
-    @Getter
-    @Setter
     private Instant modifiedDate;
 
-    public ExpensesTrackerItem() {}
-
-    public ExpensesTrackerItem(String description, Integer price,String category) {
+    public ExpensesTrackerItem(String description, Integer price, String category) {
```
Comment (samadpls) [inline src/main/java/com/scad/expensestracker/models/ExpensesTrackerItem.java:46]: Why getter setter is removed??

File: src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java (hunk 1 above)
Comment (Myrausman) [reply src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java]: that array previously was empty , i initialized list items by fetching all items n expenses list

File: src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java (hunk 2 above)
Comment (Myrausman) [reply src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java]: its a mistake , i added it but then due to some issues of git push and git pull. it got mixed up

**Review by samadpls — APPROVED** [review]
LGTM

**Comment by Myrausman** [comment]
> LGTM

Ok,ThankYou

**Pull request merged**